added sandbox links to product sections (#4370)
getsentry/sentry-docs: src/docs/product/dashboards/index.mdx

description: "Sentry's Dashboards provide a single view of your application's health, including the ability to navigate issues and performance across multiple projects."
---

> <SandboxLink scenario="dashboards" projectSlug="react">Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Reviewer: @Neo-Zhixing do you have a solution for this one? This is another case where the HTML tags in the Sandbox link are breaking the link for the markdown. Ideally, we don't use html links between the docs themselves, rather just these short links.

Reviewer 2: I think gatsby-mdx is interpreting `<SandboxLink>` as a block-level component making the links render incorrectly.
There are two workarounds:
- Don't use `<SandboxLink>` as the first thing in a paragraph. Put something else before it.
- Use `<a>` tags for links.

Obviously this is not ideal, so if it becomes a huge problem in the future, let me know and I'll go into the upstream packages to fix it.

Reviewer: I've thrown a word in front of the link so hopefully that will fix this.

Reviewer: Also @Neo-Zhixing, I'd venture to say this behaviour is already a problem (is a real problem in lists of links). Can I log it as an issue and assign it to you?

Reviewer 2: Sure, put it on growth backlog and we will revisit this in the future.

> <SandboxLink scenario="dashboards" projectSlug="react">Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Reviewer: ```suggestion
Sentry's <SandboxLink scenario="dashboards" projectSlug="react">Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
```

> <SandboxLink scenario="dashboards" projectSlug="react">custom Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Author: hm... i can remove the custom bit, we only have one dashboard section in the product and the content is referencing https://try.sentry-demo.com/organizations/viable-elk/dashboards/ which is where i'm sending the link. so i think it is still relevant.

> <SandboxLink scenario="dashboards" projectSlug="react">custom Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Reviewer: Is the sandbox going to let them customize a dashboard?

> <SandboxLink scenario="dashboards" projectSlug="react">custom Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Reviewer: ```suggestion
[Dashboards](https://try.sentry-demo.com/demo/start/?scenario=dashboards&projectSlug=react&source=docs) provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
```

> <SandboxLink scenario="dashboards" projectSlug="react">custom Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).

Reviewer: ```suggestion
<SandboxLink scenario="dashboards" projectSlug="react">Dashboards</SandboxLink> provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
```

## Open Dashboard Widgets in Discover

> Each <SandboxLink scenario="dashboards" projectSlug="react">custom Dashboard</SandboxLink> widget has an ellipsis that opens a context menu. From here, you can open the widget in [Discover](/product/discover-queries/) for to get more information. If a widget contains more than one query, you'll be prompted to select which query to view in **Discover**.

Reviewer: Same comment as above.

```suggestion
Each Dashboard widget has an ellipsis that opens a context menu. From here, you can open the widget in [Discover](/product/discover-queries/) for to get more information. If a widget contains more than one query, you'll be prompted to select which query to view in **Discover**.
```
